client/src/main/scala/drt/client/services/handlers/SimulationHandler.scala

```diff
@@ -0,0 +1,16 @@
+package drt.client.services.handlers
+
+import diode.{ActionResult, ModelRW}
+import diode.data.Pot
+import drt.client.actions.Actions._
+import drt.shared.SimulationResult
+import org.scalajs.dom._
+
+class SimulationHandler[M](simulationResult: ModelRW[M, Pot[SimulationResult]]) extends LoggingActionHandler(simulationResult) {
```

Comment:
Does this need to exist if it's not doing anything? Maybe I've missed something

Reply:
I forgot I'd added that - it may be worth keeping as we'll need a starting point when we build the next step and it will save somebody 10 minutes if this is already there.

Reply:
:+1: